.docker/gui-docker

```diff
@@ -1,11 +1,17 @@
 #!/bin/bash -u
 
 # This script is used to run a docker container with graphics support.
+# Documentation: https://moveit.ros.org/install/docker/
 # http://wiki.ros.org/docker/Tutorials/Hardware%20Acceleration
 
-# All arguments to this script will be appended to a docker run command.
-# Example command line:
-# ./gui-docker -it --rm moveit/moveit:master-source /bin/bash
+# Runs and enters a persistent docker container with moveit installed by default.
+# All arguments to this script except "-c" will be appended to a docker run command.
+
+# Example commands:
+# ./gui-docker
+# ./gui-docker -c different_container_name                              # To create a different persistent container
+# ./gui-docker --rm -it moveit/moveit:master-source /bin/bash           # To remove and recreate the container
```

Comment:
This command creates a new, randomly named container, which is removed on exit.

Reply:
True, I misread. Will update the other PR as well.